contracts/identity/Identity.sol

```diff
@@ -223,7 +223,8 @@ contract Identity is IdentityAdminRole, SchemeGuard, Pausable {
      * @param account the address to add
      */
     function _addWhitelisted(address account) internal {
-        whitelist.add(account);
+        if (!whitelist.has(account))
```

Comment:
this is very wrong. remove it. if user is whitelisted he can't be added again
same for removewhitelisted

Reply:
it was a draft and not a final version, just for testing. it was replaced by using `authenticate`.
db43a7633912fed7aa89a7ff580fbe5238d650b5